apps/datahub/src/styles.css

```diff
@@ -29,3 +29,9 @@ html.record-page-active {
     scroll-padding-top: 80px;
   }
 }
+
+@layer components {
+  .badge-btn {
+    @apply items-center justify-center px-2 py-1 bg-primary hover:text-primary text-white rounded transition-all;
+  }
+}
```

Comment:
global CSS :scream: I thought we weren't doing that any more! why not specify this in the datahub header CSS at least?

Reply:
Hold on !